examples/with-sentry/pages/_error.js

```diff
@@ -1,12 +1,13 @@
 import NextErrorComponent from 'next/error'
 import * as Sentry from '@sentry/node'
 
-const MyError = ({ statusCode, hasGetInitialPropsRun, err }) => {
+const MyError = async ({ statusCode, hasGetInitialPropsRun, err }) => {
   if (!hasGetInitialPropsRun && err) {
     // getInitialProps is not called in case of
     // https://github.com/vercel/next.js/issues/8592. As a workaround, we pass
     // err via _app.js so it can be captured
     Sentry.captureException(err)
+    await Sentry.flush(2000)
```

Comment:
@ceorourke can you add a comment why we need flush?

Reply:
`flush` is needed in serverless applications after calling `captureException` to send server side errors to Sentry, otherwise the serverless function will exit before it's sent.

Reply:
@ceorourke No, I mean add the comment in the code lol